### PR TITLE
Add aidatatang_200zh recipe

### PR DESCRIPTION
**Pull request by nzhoward**

### CODE REVIEW

File: egs/aidatatang_200zh/asr1/conf/tuning/decode_pytorch_transformer.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+batchsize: 0
+beam-size: 10
+penalty: 0.0
+maxlenratio: 0.0
+minlenratio: 0.0
+ctc-weight: 0.5
+lm-weight: 0.7
```
Comment (sw005320) [inline egs/aidatatang_200zh/asr1/conf/tuning/decode_pytorch_transformer.yaml:7]: I feel that `lm-weight` is a bit too large.
Maybe you can tune `ctc-weight` and `lm-weight` a bit with the dev set (and probably `ngram-weight` as well)

File: egs/aidatatang_200zh/asr1/conf/tuning/train_pytorch_transformer.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+grad-clip: 5
+patience: 0
+epochs: 50
+dropout-rate: 0.1
+
+# transformer specific setting
+backend: pytorch
+model-module: "espnet.nets.pytorch_backend.e2e_asr_transformer:E2E"
+transformer-input-layer: conv2d     # encoder architecture type
+transformer-lr: 1.0
```
Comment (sw005320) [inline egs/aidatatang_200zh/asr1/conf/tuning/train_pytorch_transformer.yaml:36]: You may tune `transformer-lr`: 2.0 (or larger).

File: egs/aidatatang_200zh/asr1/run.sh [hunk 3]
```diff
@@ -0,0 +1,268 @@
+#!/bin/bash
+
+# Copyright 2017 Johns Hopkins University (Shinji Watanabe)
```
Comment (sw005320) [inline egs/aidatatang_200zh/asr1/run.sh:3]: Put your name!

File: egs/aidatatang_200zh/asr1/conf/tuning/decode_pytorch_transformer.yaml (hunk 1 above)
Comment (nzhoward) [reply egs/aidatatang_200zh/asr1/conf/tuning/decode_pytorch_transformer.yaml:7]: Should I try tuning this in a later PR?

File: egs/aidatatang_200zh/asr1/conf/tuning/train_pytorch_transformer.yaml (hunk 2 above)
Comment (nzhoward) [reply egs/aidatatang_200zh/asr1/conf/tuning/train_pytorch_transformer.yaml:36]: Same with above.

File: egs/aidatatang_200zh/asr1/conf/tuning/decode_pytorch_transformer.yaml (hunk 1 above)
Comment (sw005320) [reply egs/aidatatang_200zh/asr1/conf/tuning/decode_pytorch_transformer.yaml:7]: yes, let's do it later.
let's try to finish this PR anyway.

File: egs/aidatatang_200zh/asr1/run.sh (hunk 3 above)
Comment (sw005320) [inline egs/aidatatang_200zh/asr1/run.sh:3]:
```suggestion
# Copyright 2020 Johns Hopkins University (Hao Yan )
```

File: egs/aidatatang_200zh/asr1/conf/tuning/decode_pytorch_transformer.yaml (hunk 1 above)
Comment (nzhoward) [reply egs/aidatatang_200zh/asr1/conf/tuning/decode_pytorch_transformer.yaml:7]: I've added the description to https://github.com/espnet/espnet/blob/master/egs/README.md

**Comment by sw005320** [comment]
Many thanks! Please make a follow-up PR after tuning.

**Pull request merged**